RELEASE.md

```diff
@@ -17,6 +17,7 @@
 
 ## Bug fixes and other changes
 * Removed fatal error from being logged when a Kedro session is created in a directory without git.
+* Fixed `CONFIG_LOADER_CLASS` validation so that `TemplatedConfigLoader` can be specified in settings.py.
```

Comment:
Might be helpful for future reference to mention how we fixed it, so something like fixed config_loader_class validation by checking for shared parent class, so that ...

Reply:
Done!